recipe/meta.yaml

```diff
@@ -51,9 +52,7 @@ requirements:
     - twisted
     - uncertainties
     - xhtml2pdf
-    - {{ pin_compatible('numpy') }}
-
-
+    - {{ pin_compatible('numpy',max_pin='x.x') }}
```

Comment:
Should this have an actual max version instead of 'x.x'?  It does seem to work so maybe doesn't matter?

Reply:
@pbeaucage Sorry, I missed it. My understanding is that values for x.x are taken from the "host" (https://github.com/conda-forge/sasview-feedstock/blob/d1e591cb22ae5b24de56e54c98efdf6030b64820/recipe/meta.yaml#L28)